### PR TITLE
Update `NEWS.md` in readiness `0.6.5` release

### PR DESCRIPTION
**Pull request by johanmalm**
Reviews welcome.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -23,6 +24,74 @@ The format is based on [Keep a Changelog]
...
+- Support on/off boolean configuration values (in addition to true, false,
+  yes and no)
+- keybinds
+  - Allow non-english based keybinds
+  - Make keybind agnostic to keyboard layout. Fixes #1069
+  - Add optional layoutDependent argument to only trigger if the
+    configured key exists in the currently active keyboard layout.
+    `<keybind key="" layoutDependent="">`
+  - Fallback on raw keysyms (as if there were no pressed modifier) for
+    bindings with not match against translated keysyms. This allows users
```
Comment (Consolatis) [inline NEWS.md:41]: "which do not match"

File: NEWS.md [hunk 2]
```diff
@@ -23,6 +24,74 @@ The format is based on [Keep a Changelog]
...
+- Fix invalid value in `<accelProfile>` falling back as "flat"
+- Fix touch bug to avoid jumping when a touch point moves off of a surface
+  Written-by: @bi4k8
+- Prevent crash with theme setting `osd.window-switcher.width: 0`.
+  Fixes #1050
+- cursor: prevent press on layer-subsurface from cancelling popup grab
+  Fixes #1030
+- Cancel cursor popup grab on mouse-press outside client itself, for
+  example on any part of the server side decoration or the desktop.
+  Fixes #949
```
Comment (Consolatis) [inline NEWS.md:77]: Suggest switching these two around, e.g. first "cancel popup grab" then "prevent press.."

**Comment by Consolatis** [comment]
Aside from the two comments, LGTM

File: NEWS.md (hunk 1 above)
Comment (johanmalm) [reply NEWS.md]: Good catch

File: NEWS.md (hunk 2 above)
Comment (johanmalm) [reply NEWS.md]: All done

**Comment by johanmalm** [comment]
Also added a couple of “Written-by:”

**Pull request merged**